java/core/src/main/java/co/worklytics/psoxy/rules/github/PrebuiltSanitizerRules.java

```diff
@@ -0,0 +1,696 @@
+                    .jsonPath("$..html_url")
+                    .jsonPath("$..description")
+                    .jsonPath("$..members_url")
+                    .jsonPath("$..repositories_url")
+                    .build())
+            .build();
+
+    static final Endpoint ORG_AUDIT_LOG = Endpoint.builder()
+            .pathTemplate("/orgs/{org}/audit-log")
+            .allowedQueryParams(orgAuditLogAllowedQueryParameters)
+            .transform(Transform.Redact.builder()
+                    .jsonPath("$..hashed_token")
+                    .jsonPath("$..business")
+                    .jsonPath("$..business_id")
+                    .jsonPath("$..transport_protocol")
+                    .jsonPath("$..transport_protocol_name")
+                    .jsonPath("$..pull_request_title")
+                    .jsonPath("$..user_agent")
+                    .build())
+            .transform(Transform.Pseudonymize.builder()
+                    .jsonPath("$..actor")
+                    .jsonPath("$..user")
+                    .build())
+            .build();
+
+    static final Endpoint ORG_TEAM_MEMBERS = Endpoint.builder()
+            .pathTemplate("/orgs/{org}/teams/{team_slug}/members")
+            .allowedQueryParams(commonAllowedQueryParameters)
+            .transforms(generateUserTransformations("."))
+            .build();
+
+    static final Endpoint REPO_COMMIT = Endpoint.builder()
+            .pathTemplate("/repos/{owner}/{repo}/commits/{ref}")
+            .allowedQueryParams(commonAllowedQueryParameters)
+            .transform(Transform.Redact.builder()
+                    .jsonPath("$..name")
+                    .jsonPath("$..url")
+                    .jsonPath("$..html_url")
+                    .jsonPath("$..message")
+                    .jsonPath("$..comments_url")
+                    .jsonPath("$..files")
+                    .build())
+            .transform(Transform.Pseudonymize.builder()
+                    .jsonPath("$.commit..email")
+                    .build())
+            .transforms(generateUserTransformations("..author"))
+            .transforms(generateUserTransformations("..committer"))
+            .build();
+
+    static final Endpoint ISSUES = Endpoint.builder()
+            .pathTemplate("/repos/{owner}/{repo}/issues")
+            .allowedQueryParams(issuesAllowedQueryParameters)
+            .transform(Transform.Redact.builder()
+                    .jsonPath("$..url")
+                    .jsonPath("$..repository_url")
+                    .jsonPath("$..labels_url")
+                    .jsonPath("$..comments_url")
+                    .jsonPath("$..events_url")
+                    .jsonPath("$..html_url")
+                    .jsonPath("$..title")
+                    .jsonPath("$..body")
+                    .jsonPath("$..description")
+                    .jsonPath("$..name")
+                    .jsonPath("$..diff_url")
+                    .jsonPath("$..patch_url")
```

Comment:
why redact all the URLs? don't seem inherently unsafe to me. concern that we might see a repo owned by an individual or something, then get a github user name?